Frontend/ui-customer/src/Flow.purs

```diff
@@ -1045,6 +1137,22 @@ homeScreenFlow = do
                       }
                 )
       homeScreenFlow
+
+    CONFIRM_FARE state -> do
+      void $ lift $ lift $ loaderText (getString STR.LOADING) (getString STR.PLEASE_WAIT_WHILE_IN_PROGRESS) -- TODO : Handlde Loader in IOS Side
+      void $ lift $ lift $ toggleLoader true
+      resp <- lift $ lift $ HelpersAPI.callApi $ Remote.makeEditLocResultConfirmReq (fromMaybe "" state.props.bookingUpdateRequestId)
```

Comment:
if `state.props.bookingUpdateRequestId` is nothing then we should'nt make the api call itself

Reply:
The check is already there in controller. So ideally it won't even happen.

Reply:
Still we can add it here as well instead of fromMaybe